viewer/src/dataModels/cad/internal/CadTransformationProvider.ts

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { SectorModelTransformation } from './sector/types';
+
+export interface CadTransformationProvider {
+  getCadTransformation(): SectorModelTransformation;
```

Comment:
Shouldn't this take `blobUrl`/some ID as parameter?

Reply:
yes, it will when it gets it from network :)

Reply:
Not sure why we add this interface without the parameter - if we won't add support yet is there any use in adding the interface at all?